datatree/tests/test_treenode.py

```diff
@@ -95,7 +95,7 @@ def test_ancestors(self):
         michael = TreeNode(children={"Tony": tony})
         vito = TreeNode(children={"Michael": michael})
         assert tony.root is vito
-        assert tony.lineage == (tony, michael, vito)
+        assert tony.parents == (michael, vito)
```

Comment:
A node's parents do not include the node itself, in opposition to `lineage`. 

Note:`lineage` is still tested in `test_lineage`